pkg/allocation/converters/converter_test.go

```diff
@@ -144,7 +144,7 @@ func TestConvertAllocationRequestToGameServerAllocation(t *testing.T) {
 		},
 		{
 			name:     "all fields are set",
-			features: fmt.Sprintf("%s=false&%s=false", runtime.FeaturePlayerAllocationFilter, runtime.FeatureStateAllocationFilter),
+			features: fmt.Sprintf("%s=false", runtime.FeaturePlayerAllocationFilter),
```

Comment:
For the `false` cases, you should just remove the tests.

Reply:
sorry I am not getting it, do I need to remove this block https://github.com/googleforgames/agones/blob/main/pkg/allocation/converters/converter_test.go#L217-L317 from the function `TestConvertAllocationRequestToGameServerAllocation`?

Reply:
Yes, and others cases below this. The `features` string is setting the features flag, but on Stable promotion the "=false" setting no longer exists (the feature no longer exists, it's fully incorporated into the product).